src/Uno.Extensions.Navigation.UI/IThemeService.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Uno.Extensions.Navigation.UI
+{
+	public interface IThemeService
+	{
```

Comment:
This will likely have to change given this discussion: https://github.com/unoplatform/uno.extensions/discussions/420

Reply:
updated inerface definition in #420